support-e2e/tests/monthlyContribution.test.ts

```diff
@@ -4,58 +4,58 @@ import { email, firstName, lastName } from "./utils/users";
 
 test.beforeEach(async ({ page, context, baseURL }) => {
   const baseUrlWithFallback = baseURL ?? "https://support.theguardian.com";
-  const pageUrl = `${baseUrlWithFallback}/uk/contribute`;
+  // We should remove the forcing into the ab test once this has been made live
+  const pageUrl = `${baseUrlWithFallback}/uk/contribute#ab-twoStepCheckoutWithNudgeBelow=variant_a`;
 
   const domain = new URL(pageUrl).hostname;
   await setTestCookies(context, firstName, domain);
-  await page.goto(pageUrl, {waitUntil: "networkidle"});
+  await page.goto(pageUrl, { waitUntil: "networkidle" });
 });
 
 test.describe("Sign up for a Recurring Contribution (New Contributions Flow)", () => {
   test("Monthly contribution sign-up with Stripe - GBP", async ({ page }) => {
-    const monthlyTab = "#MONTHLY";
-    await page.locator(monthlyTab).click();
+    await page.getByRole("tab").getByText("Monthly").click();
+    await page.getByText("Continue to checkout").click();
```

Comment:
Why is this a page.getByText instead of  await page.getByRole('button', { name: 'Continue to checkout' }).click() ? Is it just to reduce the code?

Reply:
Nice suggestion, I wasn't aware of that syntax and have updated it.